### DISCUSSION TITLE
Can not install via go install

### DISCUSSION THREAD

**Discussion by ghost** [j178/leetgo#105] [General]
install leetgo via [documnet](https://github.com/j178/leetgo#install-via-go) but face this problem:

```shell
~ go install github.com/j178/leetgo@latest
go: downloading github.com/j178/leetgo v0.2.0
go: github.com/j178/leetgo@latest (in github.com/j178/leetgo@v0.2.0):
        The go.mod file for the module providing named packages contains one or
        more replace directives. It must not contain directives that would cause
        it to be interpreted differently than if it were the main module.
```

here is my go env:
```shell
~ go version
go version go1.20 linux/amd64
~ go env GO111MODULE          
on
~ go env GOPROXY           
https://goproxy.cn
```

i can only install this way:

```shell
git clone git@github.com:j178/leetgo.git

cd leetgo && go install
```

find a similar case in [stackoverflow](https://stackoverflow.com/questions/69807151) 

is this a bug or golang limit?

**Comment by j178** [comment]
Thank you for your feedback. The "go install" indeed doesn't work due to the use of the "replace" directive in the go.mod. This is a limitation of "go install". For more information, see this [issue](https://github.com/golang/go/issues/44840) and [this](https://pkg.go.dev/cmd/go#hdr-Compile_and_install_packages_and_dependencies). I think I may need to remove the "replace" directive as a workaround for this issue.
